tests/cookbook_sinking_anhydrite_block.prm

```diff
@@ -0,0 +1,115 @@
+  set Model name = box
+
+  subsection Box
+    set X extent  = 2500
+    set Y extent  = 5000
+    set Y repetitions = 2
+  end
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left, right, bottom, top
+end
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 9.81
+  end
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top
+  set List of model names = box
+end
+
+subsection Initial temperature model
+  set Model name = function
+
+  subsection Function
+    set Function expression = 0
+  end
+end
+
+subsection Mesh refinement
+  set Initial adaptive refinement   = 1
+  set Initial global refinement     = 5
+  set Refinement fraction           = 0.99
+  set Strategy                      = composition
+  set Coarsening fraction           = 0.01
+end
+
+
+subsection Material model
+  set Model name = multicomponent
+  set Material averaging = harmonic average
+
+  subsection Multicomponent
+    set Densities = 0, 2200, 2200, 2900
+    set Viscosities = 1e30, 1e16, 1e17, 1e20
+    set Viscosity averaging scheme = harmonic
+    set Thermal expansivities = 0
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 3
+  set Names of fields = salt_left, salt_right, anhydrite_block
+  set Compositional field methods = particles, particles, particles
+  set Mapped particle properties = salt_left:initial salt_left, salt_right:initial salt_right, anhydrite_block: initial anhydrite_block
+end
+
+subsection Initial composition model
+  set Model name = function
+
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = bx=100,by=100,Lx2=1250,h=100,Ly=5000
+    set Function expression = if((x<=Lx2 && (y>=Ly-h || y<=Ly-h-by) || (x<=Lx2-bx/2 && y<=Ly-h && y>= Ly-h-by)),1,0) ;\
+                              if((x>Lx2 && (y>=Ly-h || y<=Ly-h-by) || (x>Lx2+bx/2 && y<=Ly-h && y>= Ly-h-by)),1,0) ;\
+                              if((abs(x-Lx2)<bx/2 && abs(y-Ly+h+by/2)<by/2),1,0)
+  end
+end
+
+subsection Postprocess
+  set List of postprocessors = visualization, velocity statistics, composition statistics, material statistics, global statistics, particles
+
+  subsection Visualization
+    set List of output variables = material properties, strain rate
+    set Time between graphical output = 0
+    set Interpolate output = false
+
+    subsection Material properties
+      set List of material properties = density, viscosity
+    end
+  end
+
+  subsection Particles
+    set Time between data output = 0
+    set Data output format       = vtu
+  end
+end
+
+subsection Particles
+  set List of particle properties = initial composition, velocity
+  set Particle generator name = random uniform
+  set Interpolation scheme = cell average
+
+  subsection Generator
+    subsection Random uniform
+      set Number of particles      = 100000
+    end
+  end
+end
```

Comment:
Can you change this to
```suggestion
include $ASPECT_SOURCE_DIR/cookbooks/sinking_of_anhydrite_block_in_salt_diapir/sinking_of_anhydrite_block_in_salt_diapir_particle_in_cell.prm

set End time                               = 0

subsection Mesh refinement
  set Initial adaptive refinement   = 1
end
```
That way, we test the actual cookbook file rather than a copy. Sorry for not making that more clear earlier.